Library/Homebrew/test/dev-cmd/audit_spec.rb

```diff
@@ -891,6 +885,29 @@ class Foo#{foo_version} < Formula
         end
       end
 
+      describe "new formulae should not have a revision" do
+        it "doesn't allow new formulae to have a revision" do
+          fa = formula_auditor "foo", <<~RUBY, new_formula: true
+            class Foo < Formula
+              url "https://brew.sh/foo-1.0.tgz"
+              revision 1
+            end
+          RUBY
+
+          fa.audit_revision_and_version_scheme
+
+          expect(fa.new_formula_problems).to include(
+            a_hash_including(message: a_string_matching(/should not define a revision/)),
+          )
+        end
+      end
+
+      subject {
```

Comment:
Is there a reason you moved this? Not a huge deal but unless there's a specific reason for moving it's probably better if it just stays where it is. Also, I think the new `describe` that you added should go after the `formula_gsub_origin_commit` method definition below.

But thanks so much for writing the test!

Reply:
Agreed to ideally leave it as-is to minimise the diff, thanks!

Reply:
Not sure I'll be able to figure this stuff out anytime soon.